app/models/event_date.rb

```diff
@@ -9,8 +9,11 @@ class EventDate < ApplicationRecord
   has_many :event_hours, foreign_key: :event_date_id, inverse_of: :event_date,
                          dependent: :restrict_with_exception
 
-  default_scope { active.published.event_publishes_dates.future.end_datetime }
-  scope :active, -> { where(status_id: 1) }
+  default_scope { active.published.event_publishes_dates.future }
+  scope :active, lambda {
+    where('event_dates.status_id = 1 AND ? <= published_end_datetime',
+          DateTime.current.utc.strftime('%Y-%m-%d %H:%M:%S'))
+  }
   scope :event_publishes_dates, lambda {
```

Comment:
On one hand, my comment feels a little silly, but on the other I think it would be best to leave active separate from 'not expired'.
e.g. 
- scope active : where status_id = 1
- scope 'not_expired' where ? <= published_end_datetime', DateTime.current.utc.strftime('%Y-%m-%d %H:%M:%S')
default scope: active.published.event_publishes_dates.future.not_expired

Our current and only use for the API means that we could certainly lump active & not expired into the same scope, but since we can easily achieve our desired result with the default scope (includes those 2 plus others) I think we should leave them separate. 

One potential upcoming use case would be to get the event_dates that were active, and occurring on a certain day. If we also applied the 'not_expired' scope inside of the active scope, that would not give us the data we wanted.